### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by thomasmfish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+  --overwrite           If specified, files will be overwritten if they
+                        already exist (unique filenames will be used
+                        otherwise)
+  --no-cleanup          If specified, files created during the OTF creation
+                        process will not be cleaned up
+  --shape XY_SHAPE XY_SHAPE
+                        Takes 2 integers (X Y), specifying the shape to crop
+                        PSFs to before converting (powers of 2 are fastest)
+  -v, --verbose         Show more logging
+  --no-progress         turn off progress bars (only has an affect if tqdm is
```
Comment (PeterC-DLS) [inline README.md]: "an effect"

File: README.md [hunk 2]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+  --wiener WIENER       Wiener constant; lower value leads to higher
+                        resolution and noise;
+  --zoomfact ZOOMFACT   lateral zoom factor in the output over the input
+                        images;
+  --zzoom ZZOOM         axial zoom factor; almost never needed
+  --background BACKGROUND
+                        camera readout background
+  --usecorr USECORR     use a flat-field correction file if provided
+  --k0angles K0ANGLES K0ANGLES K0ANGLES
+                        user these pattern vector k0 angles for all
```
Comment (PeterC-DLS) [inline README.md]: "use "

File: README.md [hunk 3]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+  --otfRA OTFRA         using rotationally averaged OTF; otherwise using
```
Comment (PeterC-DLS) [inline README.md]: incomplete sentence

File: README.md [hunk 4]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+  --otfPerAngle OTFPERANGLE
+                        using one OTF per SIM angle; otherwise one OTF is
```
Comment (PeterC-DLS) [inline README.md]: Inconsistent tense and incomplete

File: README.md [hunk 5]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+  --fastSI FASTSI       SIM image is organized in Z->Angle->Phase order;
```
Comment (PeterC-DLS) [inline README.md]: ?

File: README.md [hunk 6]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+                        bleach correction for z
+  --equalizet EQUALIZET
+                        bleach correction for time
+  --dampenOrder0 DAMPENORDER0
+                        dampen order-0 in final assembly; do not use for 2D
+                        SIM; good choice for high-background images
+  --nosuppress NOSUPPRESS
+                        do not suppress DC singularity in the result (good
+                        choice for 2D/TIRF data)
+  --nokz0 NOKZ0         not using kz=0 plane of the 0th order in the final
```
Comment (PeterC-DLS) [inline README.md]: "omit kz=0 plane..."

File: README.md [hunk 7]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+                        assembly (mostly for debug)
+  --gammaApo GAMMAAPO   output apodization gamma; 1.0 means triangular
+  --explodefact EXPLODEFACT
+                        artificially exploding the reciprocal-space
+  --nofilterovlps NOFILTEROVLPS
```
Comment (PeterC-DLS) [inline README.md:209]: Missing descrption

File: README.md [hunk 8]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+                        dampen order-0 in final assembly; do not use for 2D
+                        SIM; good choice for high-background images
+  --nosuppress NOSUPPRESS
+                        do not suppress DC singularity in the result (good
+                        choice for 2D/TIRF data)
+  --nokz0 NOKZ0         not using kz=0 plane of the 0th order in the final
+                        assembly (mostly for debug)
+  --gammaApo GAMMAAPO   output apodization gamma; 1.0 means triangular
+  --explodefact EXPLODEFACT
+                        artificially exploding the reciprocal-space
```
Comment (PeterC-DLS) [inline README.md]: Tense

File: README.md [hunk 9]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+  --nofilterovlps NOFILTEROVLPS
+  --deskew DESKEW       Deskew angle; if not 0.0 then perform deskewing before
+                        processing
+  --deskewshift DESKEWSHIFT
+                        If deskewed, the output image's extra shift in X
```
Comment (PeterC-DLS) [inline README.md]: "...use this extra shift ..."

File: README.md [hunk 10]
```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
...
+                        (positive->left)
+  --noRecon NORECON     No reconstruction will be performed; useful when
+                        combined with "deskew":
+  --cropXY CROPXY       Crop the X-Y dimension to this number; 0 means no
```
Comment (PeterC-DLS) [inline README.md:234]: pixels?

File: src/sim_recon/main.py [hunk 11]
```diff
@@ -72,6 +90,26 @@ def sim_reconstruct(
...
+    output_directory : str | PathLike[str]
+        Directory to save reconstructions in (reconstructions will be saved with the data files if not specified), by default None
+    overwrite : bool, optional
+        Overwrite files if they already exist, by default False
+    cleanup : bool, optional
+        Clean up temporary directory and files after reconstruction, by default True
+    stitch_channels : bool, optional
+        Stitch channels back together after processing (otherwise output will be a separate DV per channel), by default True
+    parallel_process : bool, optional
+        Run reconstructions in 2 processes simultaneously, by default False
```
Comment (PeterC-DLS) [inline src/sim_recon/main.py]: concurrently

File: README.md (hunk 10 above)
Comment (thomasmfish) [reply README.md:234]: I assume so, but most of this is from the cudasirecon docs, so I don't want to make assumptions

File: README.md (hunk 7 above)
Comment (thomasmfish) [reply README.md:209]: Yep, I have no idea what that one is.

**Comment by thomasmfish** [comment]
I've done my best with the descrptions of arguments from cudasirecon.

File: README.md (hunk 5 above)
Comment (PeterC-DLS) [reply README.md]: "... otherwise assume ..."

File: README.md [hunk 12]
```diff
@@ -18,16 +18,225 @@ To reconstruct the SIM data and create OTFs from PSFs from the cryoSIM at B24, D
...
+  --zoomfact ZOOMFACT   Lateral zoom factor in the output over the input
+                        images; leaving it at 2 should be fine in most cases
+  --zzoom ZZOOM         Axial zoom factor; almost never needed
+  --background BACKGROUND
+                        Camera readout background
+  --usecorr USECORR     Use a flat-field correction file if provided
+  --k0angles K0ANGLES K0ANGLES K0ANGLES
+                        Use these pattern vector k0 angles for all directions
+                        (instead of inferring the rest angles from angle0)
+  --otfRA OTFRA         Use rotationally averaged OTF; otherwise uses 3/2D OTF
```
Comment (PeterC-DLS) [inline README.md]: "otherwise use ..."

File: README.md [hunk 13]
```diff
@@ -18,16 +18,225 @@ To reconstruct the SIM data and create OTFs from PSFs from the cryoSIM at B24, D
...
+  --nosuppress NOSUPPRESS
+                        Do not suppress DC singularity in the result (good
+                        choice for 2D/TIRF data)
+  --nokz0 NOKZ0         Do not use kz=0 plane of the 0th order in the final
+                        assembly (mostly for debug)
+  --gammaApo GAMMAAPO   Output apodization gamma; 1.0 means triangular apo;
+                        lower value means less dampening of high-resolution
+                        info at the trade-off of higher noise
+  --explodefact EXPLODEFACT
+                        Artificially exploding the reciprocal-space distance
```
Comment (PeterC-DLS) [inline README.md]: Tense

Comment (thomasmfish) [reply README.md]: This was already "Artificially explode"

Comment (PeterC-DLS) [reply README.md]: Not in this README

**Comment by thomasmfish** [comment]
I've gone through the values given [in the C++](https://github.com/scopetools/cudasirecon/blob/97af94df03911735cc3ace81caf29ca36e153de0/src/cudaSirecon/cudaSirecon.cpp#L913-L1008) and have corrected various things to do with that. This has meant I was able to settings I wasn't previously sure about from the docs too.

File: README.md (hunk 13 above)
Comment (thomasmfish) [reply README.md]: Ah, I was looking at settings/formatting.py

**Review by PeterC-DLS — APPROVED** [review]
LGTM

**Pull request merged**